pages/agent/v3/tracing.md.erb

```diff
@@ -0,0 +1,24 @@
+# Tracing in the Buildkite Agent
+
+Distributed tracing tools like Datadog APM or OpenTelemetry tracing allow you to gain more insight in the the performance of your CI runs - what's fast, what's slow, what could be optimized, and more importantly, how these things are changing over time.
+
+The Buildkite Agent currently supports the two tracing backends listed above, Datadog APM (via OpenTracing) and OpenTelemetry. This doc will guide your through setting up tracing using either of these backends.
+
+{:toc}
+
+## Using Datadog APM
+
+To use the Datadog APM integration, start the Buildkite agent with the `--tracing-backend datadog` option. This will enable Datadog APM tracing, and send the traces to a Datadog Agent at `localhost:8126` by default. If your Datadog Agent is located at another host, the Buildkite Agent will respect the `DD_AGENT_HOST` and `DD_AGENT_TRACE_PORT` environment variables defined by [`dd-trace-go`](https://docs.datadoghq.com/tracing/setup_overview/setup/go/?tab=containers). Note that there will need to be a Datadog agent present at the above address to ingest these traces.
+
+Once this is done, and the will start sending tracing information to Datadog. You can observe traces as they come in in the APM > Traces screen in your Datadog instance.
+
+## Using OpenTelemetry Tracing
+
+<div class="Docs__wip-note">
+  <p class="Docs__wip-heading">Experimental feature</p>
+  <p>OpenTelemetry Tracing is an experimental feature, meaning that the implementation or API might change at any point, with no compatibility guarantees. We'll do our best to make things easy and good, but use at your own risk.</p>
+</div>
+
+To use OpenTelemetry Tracing, you'll need to enable the `opentelemetry-tracing` experiment, as well as specifying `--tracing-backend opentelemetry`.
+
+The Buildkite Agent's OpenTelemetry implementation uses the OTLP gRPC exporter to export trace information. This means that there must be a collector capable of ingesting OTLP gRPC traces accessible by the buildkite agent. By default, the Buildkite Agent will export trace information to `https://localhost:4317`. This can be overridden by passing in an environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` containing the endpoint for the collector.
```

Comment:
```suggestion
The Buildkite agent's OpenTelemetry implementation uses the OTLP gRPC exporter to export trace information. This means that there must be a [collector](/docs/test-analytics/ruby-collectors) capable of ingesting OTLP gRPC traces accessible by the Buildkite agent. By default, the Buildkite agent will export trace information to `https://localhost:4317`. This can be overridden by passing in an environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` containing the endpoint for the collector.
```

Reply:
Ah, unfortunately the collector mentioned here is a different kind of collector from the Test Analytics collectors - i'll update with your wording changes here, and then add a note into this sentence disambiguating between Otel Collectors and Test Analytics Collectors